packages/core/src/transport/httpRequest.ts

```diff
@@ -47,6 +47,18 @@ export function createHttpRequest(endpointBuilder: EndpointBuilder, bytesLimit:
     }
   }
 
+  /**
+   * Request can throw
+   * cf https://developer.mozilla.org/en-US/docs/Web/API/Request/Request#errors
+   */
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏this comment is not a function documentation, move it inside the function